m4_sessions/1708-inning/group_one.md

```diff
@@ -1,9 +1,32 @@
 # Group One
 
 ### Session Title:
+* Transhumanism - 
 
 ### Session Outcomes: 
+* Create an understanding of what transhumanism is.
+* How does this relate to us as developers of software?
+* History, now, and future of transhumanism
+* Discussion humanism and see how people's feelings and thoughts change after the session.
 
 ### Session Outline:
+* Whiteboard spectrum of how people feel about this.
+  - Have everyone go one by one, introduce themself and point out where they are in the spectrum.
+* Introduction to material
+  - History, current
+* small group discussions
+  - After we have groups discuss a main point or two, we have another pulse check on what people thing of transhumanism.
```

Comment:
What types of questions can you add to this discussion?  Be ready with thoughtful prompts.